### PR TITLE
Print random parrot with TTC_SAY_BOX=random

### PR DESCRIPTION
**Pull request by plutov**
* Add new available option for TTC_SAY_BOX
 * If it's set print random parrot every time

### CODE REVIEW

File: care.js [hunk 1]
```diff
@@ -311,18 +311,25 @@ function catSay(text) {
...
+  // Choose random parrot every time
+  if (say === 'random') {
+    var available = ['parrot', 'bunny', 'llama', 'cat', 'yeoman', 'mario', 'ironman', 'minions', 'panda'];
+    say = available[Math.floor(Math.random()*available.length)];
+  }
 
   // If config.say is custom art file path, then return custom art
-  if (artFileRegex.test(config.say)) {
-    return ansiArt.get({ filePath: config.say, speechText: textToSay });
+  if (artFileRegex.test(say)) {
```
Comment (notwaldorf) [inline care.js]: But this will never be true, right? You already overwritten it to be one of the 9 hardcoded options

File: config.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var config = {
   twitter: (process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot').toLowerCase().split(','),
 
   // Use this to have a different animal say a message in the big box.
-  say: (process.env.TTC_SAY_BOX || 'parrot').toLowerCase(),
+  say: (process.env.TTC_SAY_BOX || 'random').toLowerCase(),
```
Comment (notwaldorf) [inline config.js]: Please keep the default to be `parrot`, not `random`. I, for example, don't set this property, and would absolutely not want a random animal :)

File: care.js (hunk 1 above)
Comment (plutov) [inline care.js]: Right. We can remove `if` or we can build list of options based on content of folder with ansi arts.

File: config.js (hunk 2 above)
Comment (plutov) [inline config.js]: Good catch!

**Comment by plutov** [comment]
@notwaldorf Please check changes.

File: care.js (hunk 1 above)
Comment (plutov) [inline care.js]: Fixed.

**Comment by plutov** [comment]
Hi @notwaldorf do you still want it or we close it?

**Comment by mojoaxel** [comment]
see #137

**Pull request closed without merging**